templates/track.html

```diff
@@ -0,0 +1,183 @@
+{% set active_page = "Demo Papers" %}
```

Comment:
unfortunately I have to copy-paste the `papers.html` but I don't have time to re-factor the code so I'll live with it.
In future, we should extract common blocks from `papers.html`, `papers_vis.html`, `track.html`.